src/tocGenerator.ts

```diff
@@ -48,8 +48,17 @@ export function generateTOC(elements: YamlModel[], packageUid: string): TocItem[
         });
 
     }
+
+    if (itemsDetails.length === 0) {
+        itemsDetails = null;
+    } else {
+        itemsDetails.splice(0, 0, {
+            name: "Overview",
+            uid: packageUid
+        });
+    }
+
     return [{
-        uid: packageUid,
```

Comment:
Do you still need to return uid and only ignore it for this particular node?

Reply:
Yes, this is package's uid, which means the root node. Currently we only ignore the uid of package level.